app/config/config.yml

```diff
@@ -69,3 +69,10 @@ surfnet_saml:
             entity_id: %saml_remote_idp_entity_id%
             sso_url: %saml_remote_idp_sso_url%
             certificate: %saml_remote_idp_certificate%
+
+open_conext_engine_block_api_client:
+    http_client:
+        base_url: 'https://engine-api.vm.openconext.org/'
+        verify_ssl: false
```

Comment:
please remove from default config (or set to true as default), safe should be the default :wink: